python/paddle/nn/layer/loss.py

```diff
@@ -2226,3 +2226,258 @@ def forward(self, input, label, variance):
             self.name,
         )
         return out
+
+
+class AdaptiveLogSoftmaxWithLoss(Layer):
+    r"""Adaptive softmax is an approximate strategy for training models with large output spaces. It is most effective when
+    the label distribution is highly imbalanced, for example in natural language modelling, where the word frequency
+    distribution approximately follows the ``Zipf's law``.
```

Comment:
既然是参考 pytorch 的文档，就抄全吧， Zipf's law 附上链接
```suggestion
    distribution approximately follows the `Zipf's law <https://en.wikipedia.org/wiki/Zipf%27s_law>`_ .
```